draft-ietf-quic-multipath.md

```diff
@@ -334,23 +334,22 @@ any of the previously validated addresses.
 
 ## Path State Management
 
-An endpoint uses PATH_STATUS frames to inform that the peer should
+An endpoint uses PATH_STANDBY and PATH_AVAILABLE frames to inform that the peer should
 send packets in the preference expressed by these frames.
-Notice that the endpoint might not follow the peer’s advertisements,
-but the PATH_STATUS frame is still a clear signal of suggestion
+Note that the endpoint might not follow the peer’s advertisements,
+but these frames are still a clear signal of suggestion
 for the preference of path usage by the peer.
```

Comment:
```suggestion
for the preference of path usage by the peer.
Each peer indicates it preference of path usage independently to the other peer.
That means path usage preferences of each peer for the same path can be different.
A sender then needs to make a sending decision taking the peer's as well as its own
preferences into account which may lead to usage of paths that have been indicated
as "standby" by the peer or non-usage of some available paths.
```

Reply:
```suggestion
for the preference of path usage by the peer.
Each peer indicates its preference of path usage independently of the other peer.
It means that peers may have different usage preferences for the same path.
Depending on the sender's decisions, this may lead to usage of paths that have been
indicated as "standby" by the peer or non-usage of some locally available paths.
```